### PR TITLE
Update customizing_cli.md

### PR DESCRIPTION
**Pull request by insafuhrmann**
Typo fixes and a small wording change (as imho the sentence does not fully fit the example (no ast extraction).

### CODE REVIEW

File: hugo/content/tutorials/customizing_cli.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ export const parseAndValidate = async (fileName: string): Promise<void> => {
 };
 ```
 
-A good amount of the contents for our custom action are shared with the `generateAction` function. This isn't surprising, given that we still need to set up our language's services, and we still need to be able to extract the AST.
+Some amount of the contents for our custom action are shared with the `generateAction` function. This isn't surprising, given that for example we still need to set up our language's services.
```
Comment (montymxb) [inline hugo/content/tutorials/customizing_cli.md:76]: I'm on board with dropping the note about extracting the AST. However I would also suggest another slight change to the 2nd sentence, dropping for example and going for a simpler sentence.
```suggestion
Some amount of the contents for our custom action are shared with the `generateAction` function. This isn't surprising given that we still need to set up our language's services.
```

**Review by montymxb — CHANGES_REQUESTED** [review]
Typo fixes look good, but small suggestion on one sentence.

**Comment by insafuhrmann** [comment]
@montymxb 
Thanks for the review, I committed the requested changes.

**Review by montymxb — APPROVED** [review]
:octocat: approved!

**Comment by insafuhrmann** [comment]
Thanks @montymxb!

**Pull request merged**